### PR TITLE
Sass setup

### PR DESCRIPTION
**Pull request by Fermain**
Installs SASS, does basic bootstrap customisation and includes a watch script to run the project.

### CODE REVIEW

File: src/scss/_variables.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
+$blue:    #4072bd;
+$indigo:  #532896;
+$purple:  #705e93;
+$pink:    #852756;
+$red:     #6d1921;
+$orange:  #91582a;
+$yellow:  #c2a85a;
+$green:   #22db31;
```
Comment (jaaneh) [inline src/scss/_variables.scss]: I think we should use a different shade of green for the site.
```suggestion
$green:   #179922;
```

**Review by jaaneh — CHANGES_REQUESTED** [review]
Looks good. I've made one suggestion of changing the shade of green used.

File: src/scss/_variables.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+$teal:    #1d6b54;
+$cyan:    #51bdd3;
+
+$spacer: 1rem;
```
Comment (Fermain) [inline src/scss/_variables.scss:12]:
```suggestion
$spacer: 2rem;
```

Hey I thought this was meant to be 2, did you change your mind?

File: src/scss/styles.scss [hunk 3]
```diff
@@ -0,0 +1,2 @@
+@import "variables";
```
Comment (Fermain) [inline src/scss/styles.scss:1]: We're also going to need `x.scss` to create the theme.

**Review by Fermain — COMMENTED** [review]
This is good but we need a couple more things.

File: src/scss/_variables.scss (hunk 2 above)
Comment (Fermain) [reply src/scss/_variables.scss:12]: Nah it was always 1, you are wrong. 1 is best.

File: src/scss/styles.scss (hunk 3 above)
Comment (Fermain) [reply src/scss/styles.scss:1]: Yeah I forgot about that, thanks! Added now.

File: src/scss/_variables.scss (hunk 1 above)
Comment (Fermain) [reply src/scss/_variables.scss]: Awesome green 10/10

**Pull request merged**